app/auth/views.py

```diff
@@ -74,3 +74,13 @@ def send_signup_email():
             status=400,
             headers=None
         )
+
+
+@auth.route('/logout', methods=['GET'])
+@login_required
```

Comment:
You either use this decorator which returns `401 unauthorized` or you do explicitly within the function and still return 401